### ISSUE TITLE
Low Priority: Scan stories from Nuxt layers

### ISSUE THREAD

**Issue by peteromano** [storybook-vue/storybook-nuxt#40]
With Nuxt's layer/extends mechanism, my idea is that folks could extend from abstract layers or apps, and the only "main" app could use Storybook to compile all components from the hierarchy into a only single SB instance.

I have a separate Nuxt module writing out the layer paths (i.e., on nuxt prepare), then use it in SB main.

Feels like storybook-vue/nuxt could easily include these paths by default instead of the user defining it:

```
    addTemplate({
      filename: 'layers.mjs',
      write: true,
      getContents () {
        // @ts-ignore
        return `export default [${nuxt.options._layers.map(({ cwd }) =>
  `
  '${cwd}'`).join(',')}\n]`
      }
    })
```

```
import layers from '../.nuxt/layers'

const config: StorybookConfig = {
  stories: layers.map(layer => [
    `${layer}/**/*.mdx`,
    `${layer}/**/*.stories.@(js|jsx|ts|tsx)`
  ]).flat()),
  ...
}
```

**Comment by chakAs3** [comment]
@peteromano  

> Feels like storybook-vue/nuxt could easily include these paths by default instead of the user defining it

yes good point, it is in my pipeline already. we will discuss it once i start working on it.
